libpod/image/image.go

```diff
@@ -244,48 +240,143 @@ func (ir *Runtime) SaveImages(ctx context.Context, namesOrIDs []string, format s
 			if isTagged {
 				iData.tags = append(iData.tags, refTagged)
 			}
+		} else {
+			iData.destNames = []string{""}
```

Comment:
This shouldn’t throw away any previous tags. Maybe add a single `imageSpecifiedByID` boolean?